labs/lab0/chapter0.md

```diff
@@ -23,22 +23,22 @@ $ ssh -i rhte.pem ec2-user@student-<number>.labs.sysdeseng.com
 
 **NOTE**: For Windows users you will have to use a terminal like [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) to SSH using the private key. 
 
-Once installed, use the following instructions to SSH to the Tower instance link: [http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/putty.html](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/putty.html).
+Once installed, use the following instructions to SSH to your VM instance: [http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/putty.html](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/putty.html).
 
-TIP: Use the rhte.ppk key located at:  [instructor host](https://instructor.labs.sysdeseng.com/rhte.ppk)
+TIP: Use the rhte.ppk key located at:  [instructor host](https://instructor.labs.sysdeseng.com/rhte.ppk) as PuTTY uses a different format for its keys.
```

Comment:
do we need to mention PuTTY at all for this lab since the students will have laptops we control?

Reply:
no, although we should add something, they just need rhel 7.5, right? i think we tell people what to install